server/init/eventConfig.js

```diff
@@ -1,7 +1,7 @@
 import moment from 'moment-timezone';
 import moment2 from 'moment';
 const config = {
-  cfpStartDate: process.env.CFP_START_DATE ||  "2020-05-01",
+  cfpStartDate: process.env.CFP_START_DATE ||  "2019-05-01",
```

Comment:
is it SAFE :) to have 2019 here?

Arn't we moving to 2020 and deploying to Production?